docs/getting-started/install-configure.md

```diff
@@ -0,0 +1,384 @@
+SA="${SA_NAME}@${PROJECT_ID}.iam.gserviceaccount.com" 
+gcloud iam service-accounts create $SA_NAME --project $PROJECT_ID
+
+# enable cloud API
+SERVICE="sqladmin.googleapis.com"
+gcloud services enable $SERVICE --project $PROJECT_ID
+
+# grant access to cloud API
+ROLE="roles/cloudsql.admin"
+gcloud projects add-iam-policy-binding --role="$ROLE" $PROJECT_ID --member "serviceAccount:$SA"
+
+# create service account keyfile
+gcloud iam service-accounts keys create creds.json --project $PROJECT_ID --iam-account $SA
+```
+
+### Create a Provider Secret
+
+```
+kubectl create secret generic gcp-creds -n crossplane-system --from-file=key=./creds.json
+```
+
+### Configure the Provider
+Create the following `provider.yaml`:
+
+```
+apiVersion: gcp.crossplane.io/v1alpha3
+kind: Provider
+metadata:
+  name: gcp-provider
+spec:
+  # replace this with your own gcp project id
+  projectID: my-project
+  credentialsSecretRef:
+    namespace: crossplane-system
+    name: gcp-creds
+    key: key
+```
+
+Then apply it:
+```
+kubectl apply -f provider.yaml
+```
+
+</div>
+<div class="tab-pane fade" id="provider-tab-azure" markdown="1">
+
+### Install Azure Provider
+
+```
+PACKAGE=crossplane/provider-azure:master
+NAME=provider-azure
+
+kubectl crossplane package install --cluster --namespace crossplane-system ${PACKAGE} ${NAME}
+```
+
+### Get Azure Principal Keyfile
+
+```
+# create service principal with Owner role
+az ad sp create-for-rbac --sdk-auth --role Owner > "creds.json"
+
+# add Azure Active Directory permissions
+AZURE_CLIENT_ID=$(jq -r ".clientId" < "./creds.json")
+
+RW_ALL_APPS=1cda74f2-2616-4834-b122-5cb1b07f8a59
+RW_DIR_DATA=78c8a3c8-a07e-4b9e-af1b-b5ccab50a175
+AAD_GRAPH_API=00000002-0000-0000-c000-000000000000
+
+az ad app permission add --id "${AZURE_CLIENT_ID}" --api ${AAD_GRAPH_API} --api-permissions ${RW_ALL_APPS}=Role ${RW_DIR_DATA}=Role
+az ad app permission grant --id "${AZURE_CLIENT_ID}" --api ${AAD_GRAPH_API} --expires never > /dev/null
+az ad app permission admin-consent --id "${AZURE_CLIENT_ID}"
+```
+
+### Create a Provider Secret
+
+```
+kubectl create secret generic azure-creds -n crossplane-system --from-file=key=./creds.json
+```
+
+### Configure the Provider
+Create the following `provider.yaml`:
+
+```
+apiVersion: azure.crossplane.io/v1alpha3
+kind: Provider
+metadata:
+  name: azure-provider
+spec:
+  credentialsSecretRef:
+    namespace: crossplane-system
+    name: azure-creds
+    key: key
+```
+
+Then apply it:
+```
+kubectl apply -f provider.yaml
+```
+
+</div>
+<div class="tab-pane fade" id="provider-tab-alibaba" markdown="1">
+
+### Install Alibaba Provider
+
+```
+PACKAGE=crossplane/provider-alibaba:master
+NAME=provider-alibaba
+
+kubectl crossplane package install --cluster --namespace crossplane-system ${PACKAGE} ${NAME}
+```
+
+### Create a Provider Secret
+
+```
+kubectl create secret generic alibaba-creds --from-literal=accessKeyId=<your-key> --from-literal=accessKeySecret=<your-secret>
```

Comment:
There might be an easier way to get these values but I don't know it immediately. They are stored in `~/.aliyun/config.json` but I think what we have here is good :+1: